docs/guides/admin/docs/configuration/basic.md

```diff
@@ -26,7 +26,7 @@ your own domain name:
 processed media. At least not without an extra amount of work involving modifications to the database. That is why you
 should think about this setting carefully.
 
-Second, adjust the binding address in `org.ops4j.pax.web.cfg`. The binding address can be set to `0.0.0.0` for general
+Second, adjust the binding address in `/etc/opencast/org.ops4j.pax.web.cfg`. The binding address can be set to `0.0.0.0` for general
```

Comment:
The problem with this change is that it's not necessarily true for all installations. E.g. for most custom installations it would be `/opt/opencast/etc/...` and it really could be anywhere else. Most of our documentation uses `etc/org.ops4j.pax.web.cfg` to indicate it's somewhere in the configuration folder. Do you think that wouzld work here as well?

This modification also breaks the style check which requires the line length to be leass than 120 characters. It would be great if you could change that as well.